src/doc/style-guide/src/items.md

```diff
@@ -366,7 +366,7 @@ where
         + Index<RangeTo<Idx>, Output = Self::Output>
         + Index<RangeFrom<Idx>, Output = Self::Output>
         + Index<RangeInclusive<Idx>, Output = Self::Output>
-        + Index<RangeToInclusive<Idx>, Output = Self::Output> + Index<RangeFull>
+        + Index<RangeToInclusive<Idx>, Output = Self::Output> + Index<RangeFull>,
```

Comment:
If "each bound should go on its own line", then shouldn't it be like this?

```suggestion
        + Index<RangeToInclusive<Idx>, Output = Self::Output>
        + Index<RangeFull>,
```

Reply:
Good catch, thank you! Not directly related to this PR, but I've folded the fix into this PR (rather than putting it in a separate PR) to avoid spurious merge conflicts.